sympy/ntheory/continued_fraction.py

```diff
@@ -136,6 +136,7 @@ def continued_fraction_reduce(cf):
     """
     from sympy.core.symbol import Dummy
     from sympy.solvers import solve
```

Comment:
Yes, thanks.

-----Original Message-----
From: "Gaurav Dhingra" <notifications@github.com>
Sent: ‎02-‎08-‎2015 14:40
To: "sympy/sympy" <sympy@noreply.github.com>
Cc: "AMiT Kumar" <dtu.amit@gmail.com>
Subject: Re: [sympy] Replace solve with solveset in sympy.ntheory (#9745)

In sympy/ntheory/continued_fraction.py:
> @@ -136,6 +136,7 @@ def continued_fraction_reduce(cf):
>      """
>      from sympy.core.symbol import Dummy
>      from sympy.solvers import solve
Is not this import to be removed ?
—
Reply to this email directly or view it on GitHub.